README.md

```diff
@@ -25,20 +25,20 @@ The first step is to create an instance of the Vantiq SDK providing the URL of t
         apiVersion: 1
     });
 
-where `<server>` is the full URL for the Vantiq server to connect to, such as *https://dev.vantiq.com/* and `apiVersion` is the version of the API to use.  If not specified, this defaults to the latest version, currently *1*.  At this point, the *Vantiq* instance has not yet connected to the server.  To establish a connection to the server, use the `authenticate` method:
+where `<server>` is the full URL for the Vantiq server to connect to, such as *https://dev.vantiq.com/* and `apiVersion` is the version of the API to use.  If not specified, this defaults to the latest version, currently *1*.  At this point, the *Vantiq* instance has not yet connected to the server.  To establish a connection to the server using a username and password, use the `authenticate` method:
```

Comment:
This may still need some additional detail.  For example, in the cloud use-case you may want to say that a) an access token is required and b) such a token is created in Modelo.  It may also be useful to know that the username/password approach is for the local/edge use-case.  I might also lead with the cloud use-case, though I'm not sure whether its more common or not when using the SDK.